crates/weaver_semconv/data/expected/any_value.yaml

```diff
@@ -0,0 +1,72 @@
+body:
+  type: map
+  id: id
```

Comment:
The only "possible" issue with the current structure is that the `id` is not always required (for simple maps) at this location, it make sense at the later (nested) levels saying hey I have this field which is a map.

But here this is "body" specific. Assuming that we make the "body" just an `any_value` which would be the other way to change this -- except it make sense for `body` to be an any_value

Reply:
We could just make it "optional" (default to empty) and then "enforce" the presence as part of resolving the schema once it loaded...

Reply:
Actually, just started rolling this into my PR and with the `AnyValueSpec` this will completly replace all of the `BodySpec` stuff -- as `body` is now just the name of the `AnyValue` 😄